### PR TITLE
removed contributors section

### PR DESCRIPTION
**Pull request by oilyn0s3**
Hey, I've removed Contributors Section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,17 +89,3 @@ console.log(fieldPrefix);
 > Note: Sample model definition is available in [data/model.json](data/model.json)
 
 
-## Contributing
```
Comment (arshadkazmi42) [inline README.md]: We do not want to remove this section `Contributing` 

can you revert this.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for taking this up.

There is one small change, we want to remove only `Contributors` section and we do not want to remove `Contributing` section.

Can you make these changes? and after that we can get this merged?

File: README.md (hunk 1 above)
Comment (oilyn0s3) [reply README.md]: okay, will do!

**Comment by oilyn0s3** [comment]
Added "Contribution" section back to readme.md

**Review by arshadkazmi42 — COMMENTED** [review]
Looks good.

**Pull request merged**